context/context.go

```diff
@@ -292,19 +291,18 @@ func (context *AMFContext) AmfUeDeleteBySuci(suci string) (ue *AmfUe, ok bool) {
 	return
 }
 
-func (context *AMFContext) NewAmfRan(conn net.Conn) *AmfRan {
+func (context *AMFContext) NewAmfRan(remoteAddr string) *AmfRan {
```

Comment:
Use gnbId instead of address.

Reply:
Keep original support of creating NewAmfRan. This helps in running AMF without sctplb